glean-core/tests/common/mod.rs

```diff
@@ -12,6 +12,20 @@ use chrono::offset::TimeZone;
 use iso8601;
 use iso8601::Date::YMD;
 
+use ctor::ctor;
+
+/// Initialize the logger for all tests without individual tests requiring to call the init code.
+/// Log ouptut can be controlled via the environment variable `RUST_LOG` for the `glean_core` crate,
+/// e.g.:
+///
+/// ```
+/// export RUST_LOG=glean_core=debug
+/// ```
+#[ctor]
+fn init() {
+    let _ = env_logger::builder().is_test(true).try_init();
```

Comment:
@badboy , should this call in [`glean_enable_logging`](https://github.com/mozilla/glean/blob/c96af194a24759b741148b4b179e6d9b30e2b63a/glean-core/ffi/src/lib.rs#L50)?

If not, why?

Reply:
Oh, I now understand. That's required due to what @mdboom mentioned in his comment, right? Still, let's add a comment here to disambiguate and mention what's the relation with the other logging function

Reply:
`glean_enable_logging` is from the FFI component. We don't have that available in the `glean_core` tests.

Reply:
@Dexterp37 	there's a big comment above the function (but actually I should rename the function). Do you think that comment is enough?

Reply:
@badboy I'm no sure I understand how the comment relates to `glean_enable_logging`, sorry :( What I'm trying to understand is: do we still need the logger to be instantiated in a platform specific way (thus calling the same code `glean_enable_logging` has) also when running unit tests? If the answer is "no", is it because we want unit tests to always print to stdout/stderrr regardless of the platform they run on? If that's correct, can you mention it in a comment?

Reply:
Ah, I didn't get that from the first comment. I'll address this.